docs/UserGuide.md

```diff
@@ -88,16 +89,22 @@ to help you with the installation. Follow the guide for your operating system fo
 
 This section briefly explains the various section of TAB's GUI.
 
-![layout](images/annotation.png)
+![layout](images/annotatedGui.png)
 <div class="caption">Basic layout of TAB's user interface.</div>
 
-No. | Section | Representation
-----|---------|---------
-1 | Command Box | The text field where you key in commands that are meant to be processed by TAB. TAB will execute the command after you pressed <kbd>Enter</kbd>.
-2 | Result Display | The area that shows the result of the execution of the command. If the command entered has been executed successfully, it will display relevant success messages, otherwise, it will show error messages indicating the cause of the error.
-3 | Student List Panel | The area that shows the list of students you have in TAB.
-4 | Lesson List Panel | The area that shows the name of the student you have selected together with a list of lessons that the student has.
-5 | Center Panel | The area that displays the lists of students and lessons, calendar or the list of tags depending on the command you have entered. See [Features](#features) for more details.
+Section | Representation
+---------|---------
+Menu Bar | The area where you can click to navigate and view some of TAB's features (e.g. help, reminder and calendar). See [Features](#features) for more details.
+Menu Item | A button that you can click to view the stated feature, or to quit TAB (for <kbd>Exit</kbd>).
+Command Box | The text field where you key in commands that are meant to be processed by TAB. TAB will execute the command after you pressed <kbd>Enter</kbd>.
+Result Display | The area that shows the result of the execution of the command. If the command entered has been executed successfully, it will display relevant success messages, otherwise, it will show error messages indicating the cause of the error.
```

Comment:
```suggestion
Result Display | The area that shows the result of the execution of the command. If the command entered has been executed successfully, it will display the relevant success message. Otherwise, it will show an error message indicating the cause of the error.
```